discord_bot_aternos.py

```diff
@@ -1,81 +1,95 @@
-import discord
-from python_aternos import Client
-from python_aternos import Status
+from python_aternos import Client,Status
 import os
-from dotenv import load_dotenv
-load_dotenv()
+import interactions
+
+#In the Discord developer portal remember to surely enable messages in privileged intents else it won't work
+
+bot = interactions.Client(os.getenv("TOKEN"))
 
-# Set up Discord client with message PRIVILEGED intents enabled in DEVELOPER PORTAL
-intents = discord.Intents(messages=True)
-intents.message_content = True
-client = discord.Client(intents=intents)
 
 
 help_message = "```Available commands:\n" \
-               "!startserver - Start the Aternos server.\n" \
-               "!serverstatus - Check the status of the Aternos server.\n" \
-               "!playercount - Check the number of players on the Aternos server.\n" \
-               "!playerlist - Check the list of players on the Aternos server.\n" \
-               "!serveraddress - Check the address of the Aternos server.```"
+               "/startserver - Start the Aternos server.\n" \
+               "/serverstatus - Check the status of the Aternos server.\n" \
+               "/playercount - Check the number of players on the Aternos server.\n" \
+               "/playerlist - Check the list of players on the Aternos server.\n" \
+               "/serveraddress - Check the address of the Aternos server.```"
+
 
-@client.event
+
+@bot.event
 async def on_ready():
     print('Bot is ready.')
 
 
-@client.event
-async def on_message(message):
-
-    # Fill in your ATERNOS CREDENTIALS in env file
+@bot.command(name="startserver",description="Starts Minecraft Server")
+async def startserver(ctx: interactions.CommandContext):
     api = Client.from_credentials(os.getenv('ATERNOS_USERNAME'), os.getenv('ATERNOS_PASSWORD'))
+    servers = api.list_servers()
+    server = servers[2]
+
+    if server.status_num == Status.on:
+        await ctx.send('Server is already **online**!')
+    elif server.status_num == Status.off:

```

Comment:
if status have only On and Off. better use if , else instead of elif
Also, try to learn and implement guard conditions to reduce the multiple if else chain